governance/votes/Executive vote - July 06, 2022.md

```diff
@@ -0,0 +1,88 @@
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Core Unit Compensation
+
+#### Core Unit Budget  Streams
+
+As per multiple successful budget proposals (linked below), a total of **3,902,998.22 DAI** will be streamed to multiple Core Units if this executive proposal passes:
+
+- **989,004 DAI** will be streamed to the [Strategic Finance Core Unit (SF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP27) at [0xf737C76D2B358619f7ef696cf3F94548fEcec379](https://etherscan.io/address/0xf737C76D2B358619f7ef696cf3F94548fEcec379) starting on 2022-07-01 and ending on 2023-07-01 as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmVMc78Z).
+- **2,913,994.22 DAI** will be streamed to the [Growth Core Unit (GRO-001)](https://mips.makerdao.com/mips/details/MIP39c2SP4) at [0x7800C137A645c07132886539217ce192b9F0528e](https://etherscan.io/address/0x7800C137A645c07132886539217ce192b9F0528e) starting on 2022-07-01 and ending on 2023-06-30 as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmdMkg2N).
+
+#### Core Unit Budget Transfers
+
+- **648,133.59  DAI** will be transferred to the [Growth Core Unit (GRO-001)](https://mips.makerdao.com/mips/details/MIP39c2SP4) at [0x7800C137A645c07132886539217ce192b9F0528e](https://etherscan.io/address/0x7800C137A645c07132886539217ce192b9F0528e) as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmdMkg2N).
+
+
+### Recognized Delegate Compensation
+
+As per [MIP61: Recognized Delegate Compensation](https://mips.makerdao.com/mips/details/MIP61), a total of **85,185 DAI** will be distributed to 14 Recognized Delegates as itemized below if this executive proposal passes:
+
+- **11573 DAI** transferred to [Feedblack Loops LLC](https://vote.makerdao.com/address/0x845b36e1e4f41a361dd711bda8ea239bf191fe95) at [0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1](https://etherscan.io/address/0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1).
+- **11,528 DAI** transferred to [Flip Flop Flap Delegate LLC](https://vote.makerdao.com/address/0xaf8aa6846539033eaf0c3ca4c9c7373e370e039b) at [0x688d508f3a6B0a377e266405A1583B3316f9A2B3](https://etherscan.io/address/0x688d508f3a6B0a377e266405A1583B3316f9A2B3).
+- **11,292 DAI** transferred to [schuppi](https://vote.makerdao.com/address/0xb21e535fb349e4ef0520318acfe589e174b0126b) at [0x89C5d54C979f682F40b73a9FC39F338C88B434c6](https://etherscan.io/address/0x89C5d54C979f682F40b73a9FC39F338C88B434c6).
```

Comment:
This is an outdated address for schuppi - guessing you copy-pasted from a previous exec?

```suggestion
- **11,292 DAI** transferred to [schuppi](https://vote.makerdao.com/address/0xb21e535fb349e4ef0520318acfe589e174b0126b) at [0xCCffDBc38B1463847509dCD95e0D9AAf54D1c167](https://etherscan.io/address/0xCCffDBc38B1463847509dCD95e0D9AAf54D1c167).
```

Reply:
Correct *shame*